src/compiler/tools/TypeChecker.ts

```diff
@@ -179,6 +180,27 @@ export class TypeChecker {
         return symbol == null ? undefined : this._context.compilerFactory.getSymbol(symbol);
     }
 
+    /**
+     * Gets the resolved signature from a node.
+     * @param node - Node to get the signature from.
+     * @returns undefined if signature cannot be resolved
+     */
+    getResolvedSignature(node: Node<ts.CallLikeExpression>): Signature | undefined {
```

Comment:
I just added the `CallLikeExpression` alias in the dev branch. You can use it here instead of `Node<ts.CallLikeExpression>`.

Reply:
fixed this in #550